ci/jenkins/test.sh

```diff
@@ -138,14 +138,16 @@ function clean_antrea {
     kubectl get pod -n kube-system -l component=antrea-agent --no-headers=true | awk '{print $1}' | while read AGENTNAME; do
         kubectl exec $AGENTNAME -c antrea-agent -n kube-system -- ovs-vsctl del-port br-int gw0 || true
     done
+    # Delete antrea-prometheus first for k8s>=1.22 to avoid Pod stuck in Terminating state.
```

Comment:
Why Pod can stuck?

Reply:
I checked k8s 1.19~1.23, and found the Pod will stuck in Terminating state if Pod is deleted without CNI and "--force" flag in K8s>=1.22 . Thus we must delete antrea-prometheus before delete antrea or add "--force" to delete antrea command in this script, and I implemented the first solution.
I think this should be a behavior change on K8s side. @tnqn Do you have any idea?

Reply:
It's caused by an implementation change made in https://github.com/kubernetes/kubernetes/pull/102344 introduced in 1.22, but I don't know whether preventing Pods from being deleted until CNI Del executes sucessfully is intended.